plugins/processors/awsapplicationsignals/internal/resolver/kubernetes.go

```diff
@@ -523,38 +481,48 @@ func getKubernetesResolver(platformCode, clusterName string, logger *zap.Logger)
 		// jitter calls to the kubernetes api
 		jitterSleep(jitterKubernetesAPISeconds)
 
-		sharedInformerFactory := informers.NewSharedInformerFactory(clientset, 0)
+		sharedInformerFactory := informers.NewSharedInformerFactoryWithOptions(clientset, 0, informers.WithTweakListOptions(func(options *metav1.ListOptions) {
+			options.ResourceVersion = ""
```

Comment:
Q: is `WithTweakListOptions` with `options.ResourceVersion = ""` necessary?

Reply:
This is mainly to reduce the memory spike on start.

Reply:
I reverted this part of the logics as it causes inconsistency in our stateful mappings.